.github/workflows/trigger-release-candidate-build-and-deploy.yml

```diff
@@ -4,10 +4,19 @@ on:
   push:
     branches:
       - main
-
+  workflow_run:
+    workflows:
+      - "Push Tests"
+    types:
+      - completed
+    branches:
+      - main
```

Comment:
Do we know that the `on:push:branches:main` _will not_ trigger this workflow _before_ the the "push tests" completes? I'm just not sure how the precedence of these two triggers works out in practice. In other words, are these two conditions evaluated as AND or OR?

Reply:
That's a good point. I can start by removing `on:push:branches:main` and see if it triggers after `Push Tests` runs on `main`. If so, I think that is sufficient.

Reply:
I interpret that a little differently. Namely that whatever version of the workflow file that is on main will be the version that is run, and it won't run an updated version of the workflow file that happens to be on the development branch.